<button ng-show="false" class="btn btn-sm" ng-mouseover="vm.enabled = true" ng-mouseleave="vm.enabled = false">Test</button>
<button ng-show="false" class="btn btn-sm" ng-click="vm.Stop()">STOP</button>
<div ng-show="vm.GazeGrid.bShowGridStatBox" class="TestButtons">
    <div ng-show="!vm.GazeGrid.bShowGridStats">
        <h3>Stare at Grid number {{vm.GazeGrid.nSelectedGridNum}} for 5 seconds</h3>
    </div>
    <div ng-show="vm.GazeGrid.bShowGridStats">
        <h3>Here are the results for Grid {{vm.GazeGrid.nSelectedGridNum}}</h3>

        <h5>Correct X coordinates <b>{{vm.Stats.rightXPer | number:2}}%</b></h5>
        <h5>Correct Y coordinates <b>{{vm.Stats.rightYPer | number:2}}%</b></h5>
        <h5>Correct XY coordinates <b>{{vm.Stats.rightXYPer | number:2}}%</b></h5>
    </div>
    <div ng-show="false">
        <button class="btn btn-sm" ng-click="vm.Play()">Play</button>
        <button class="btn btn-sm" ng-click="vm.Pause()">Pause</button>
        <button class="btn btn-sm" ng-click="vm.Resume()">Resume</button>
        <button class="btn btn-sm" ng-click="vm.Stop()">Stop</button>
    </div>
</div>
<div class="ClickArea" ng-click="vm.Helper.ClickAreaOnClick()" ng-style="{'left': vm.MyData.x, 'top': vm.MyData.y}" ng-class="{'ClickAreaShow' : vm.ClickArea.status == 'show', 'ClickAreaClicked' : vm.ClickArea.clicked == 'clicked'}"></div>
<div ng-repeat="i in vm.data">
    X -- {{i.x}} -- Y {{i.y}}
</div>
<div class="GA_Container">
    <div class="ImageSetContainer center-block" ng-show="vm.GazeGrid.bShow">
        <!-- Solutions -->
        <table class="table table-bordered" id="GazeGrid">
            <tbody>
                <tr ng-repeat="row in vm.GazeGrid.gridData track by $index" class="row">
                    <td ng-repeat="col in row track by $index" class="col-md-3">
                        {{col}}
                    </td>
                </tr>
            </tbody>
        </table>
    </div>
</div>
<!--
<h1>About</h1>
<p>The Michaelson lab investigates how variation in the genome affects the development and function of the mind. We take a systems biology approach to studying neurodevelopmental conditions, and we have extramurally-supported research programs involving computational methodology, animal models, and human subjects research.
    <br>
    <br>Currently, our research focal areas are:
    <br>
    <br><strong>Non-coding genetic variation in brain development and function</strong>: the goal of this NIMH-supported project is to develop software tools that enable the estimation of when and where in the development of the brain non-coding genetic variants are likely to have an impact. Currently, non-coding genetic variation remains an important but poorly understood aspect of human medical genetics. Our efforts in this area will help to illuminate the role of non-coding variation in human brain development and help scientists better interpret genomic data.</p>
<p><strong>Genetics of language impairment</strong>: in this NIDCD/NICHD-supported project, we are performing whole-genome sequencing of a cohort of Iowa children with language impairment (collected by UI collaborator Bruce Tomblin), where we hope to identify genes and mutations involved in the acquisition and use of language. We hope that research in this area will eventually help to further refine the categorization of language pathology so that therapies can be more targeted and effective.</p>
<p><strong>Transcriptional and pharmacological regulation of hippocampal neurogenesis</strong>: In this BBR Foundation-supported project, we use mouse models to investigate the role of the transcription factors NPAS3 (implicated in human mental illness, including schizophrenia) and NPAS1 in adult hippocampal neurogenesis using RNA-seq and ChIP-seq. In addition, we are investigating the transcriptome-wide response induced by treatment with the neuroprotective compound P7C3. The findings of this project will yield a better understanding of the on and off-target effects of the drug P7C3, as well as its mechanism of action, and a better understanding of the NPAS3 regulatory cascade. This project is being performed in close collaboration with the lab of Andrew Pieper.
    <br>
    <br>Other areas of interest include machine learning in the context of genomics, sex-by-genotype interactions and sex-specific effects in neurodevelopment, and translational genetics. We are constantly forming on and off-campus collaborations to further these and other areas of interest.
    <br>
    <br>In addition to our research in the lab, we are currently recruiting subjects with a wide range of neurodevelopmental conditions (including autism spectrum disorder, language impairment, developmental delay, intellectual disability, and pediatric epilepsy) under the devGenes project. See <a href="http://www.devgenes.org">www.devgenes.org</a> for more details.&nbsp;&nbsp;</p>
<p>Students and trainees in our lab generally have a background in or appreciable exposure to genetics, and a strong desire and aptitude to leverage high-performance computing power to come to a better understanding of genome biology.</p>
<p><a title="Jacob J. Michaelson, PhD, professional profile" href="http://www.medicine.uiowa.edu/dept_primary_apr.aspx?appointment=Psychiatry&amp;id=jmichaelson">Jacob J. Michaelson, PhD, professional profile</a></p>
</div>
-->
